frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvForm.tsx

```diff
@@ -90,12 +90,12 @@ const InfraEnvForm: React.FC<InfraEnvFormProps> = ({ control, handleChange }) =>
                             <CardBody>
                                 <Stack hasGutter>
                                     <StackItem>
-                                        Once you've successfully created your infrastructure environment go to the
-                                        details view and add hosts to it.
+                                        Once you successfully create your infrastructure environment, open the details
+                                        view and add hosts to it.
```

Comment:
according to BZ, this should be
```suggestion
                                        After your infrastructure environment is successfully created, open the
details view and click the "Add hosts" button.
```

Reply:
It's true. Changed!